.github/workflows/commit-checker.yml

```diff
@@ -0,0 +1,38 @@
+name: Commit checker
+
+on:
+  pull_request:
+
+jobs:
+  commit-checker:
+    name: Commit checker
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+    - name: Rebase to ${{ github.base_ref }} branch
+      run: |
+        git config user.email 'info@openttd.org'
+        git config user.name 'OpenTTD CI'
+        git rebase origin/$GITHUB_BASE_REF
+    - name: Checkout commit-checker
+      uses: actions/checkout@v1
+      with:
+        repository: OpenTTD/OpenTTD-git-hooks
+        path: git-hooks
+        ref: master
```

Comment:
Possibly we should move this to https://github.com/OpenTTD/actions, but that is future work